### PR TITLE
Remove master from ci.yml

### PR DESCRIPTION
**Pull request by dseefeld**

### CODE REVIEW

File: azure-pipelines/builds/ci.yml [hunk 1]
```diff
@@ -2,15 +2,13 @@ trigger:
   batch: true
   branches:
     include:
-    - master
     - main
     - release/*
     - dev/source-build-5.0
```
Comment (MichaelSimons) [inline azure-pipelines/builds/ci.yml:7]: Is this still applicable?  I didn't find this branch.

**Review by MichaelSimons — APPROVED** [review]
LGTM - please link the PR to the issue for the master to main rename in order to give it some context.

File: azure-pipelines/builds/ci.yml (hunk 1 above)
Comment (MichaelSimons) [reply azure-pipelines/builds/ci.yml:7]: This might be a good opportunity to clean it up if it isn't needed anymore.

**Comment by dseefeld** [comment]
For https://github.com/dotnet/source-build/issues/2094

**Review by crummel — APPROVED** [review]

File: azure-pipelines/builds/ci.yml (hunk 1 above)
Comment (dseefeld) [reply azure-pipelines/builds/ci.yml]: Thanks.  Fixed.

**Pull request merged**